docs/docs/cloud.md

```diff
@@ -1,40 +1,65 @@
----
-title: "Overview"
----
-
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Marathon Cloud is a cloud testing infrastructure built on top of the Marathon test runner.
-It automatically provisions virtual devices to accommodate your tests within 15 minutes.
-The test execution is then delegated to Marathon test runner, which handles tasks such as batching, 
-sorting, preventive retries, and post-factum retries. 
-This ensures an even distribution of tests across the provisioned devices. 
+# Introduction
+
+[Marathon Cloud](marathonlabs.io) revolutionizes your app testing experience. The platform provides infinite virtual devices and will automatically shard, sort, distribute, and retry your tests, allowing all tests to be completed in **a maximum of 15 minutes**. First **50 hours** are free!
+
+## Getting started
+
+### Install
+
+The installation can be performed using [Homebrew](https://brew.sh/). Here's how to add the MarathonLabs repository:
```

Comment:
You can use `brew install malinskiy/tap/marathon-cloud`. It adds the tap automatically